Sources/Target/Linux/Process.cpp

```diff
@@ -215,17 +215,57 @@ ErrorCode Process::wait() {
 
     switch (_currentThread->_stopInfo.event) {
     case StopInfo::kEventNone:
-      // If the thread is stopped for no reason, it means the debugger (ds2)
-      // sent a SIGSTOP to it while it was already stopped for another reason,
-      // the SIGSTOP was queued, and now we're getting this notification. We
-      // have to ignore this and just let the thread continue.
-      // If `stepping` is true, it means the thread was actually being
-      // single-stepped before stopping, so instead of doing a `resume()`, we
-      // have to do a new `step()`.
-      if (stepping) {
+      // If _stopInfo.event is kEventNone it means the thread is either
+      // 1. stopped because another thread of the inferior hit a breakpoint
```

Comment:
Technically, it's because another thread of the inferior stopped for any reason, received a signal, etc.

I'd probably word this as "another thread of the inferior stopped (e.g.: because of a breakpoint, or received a signal, etc)"/